conf.py

```diff
@@ -35,7 +35,7 @@ def get_keybinds():
     """Get the implemented keybinds, and return a dict of
     {tool: [(full_command, keybinding, context), ...]}.
     """
-    with open('dfhack.init-example') as f:
```

Comment:
Might be more technically correct to scan `data/init/*.init` for keybinding commands.

Reply:
That's a fair point. I was planning specifically on defining that file (data/init/dfhack.keybindings.init) as **the** place for default keybindings, but it's true that we could just not have a policy and pick them up from any file in the directory.

Reply:
done